src/solvers/smt2/smt2_conv.cpp

```diff
@@ -1257,7 +1257,13 @@ void smt2_convt::convert_expr(const exprt &expr)
   }
   else if(expr.id()==ID_mod)
   {
-    convert_mod(to_mod_expr(expr));
+    // TODO: change to use term in some capacity
+    (void)convert_mod(to_mod_expr(expr));
+
+    convert_expr(expr.op0());
```

Comment:
There shouldn't be any need for this, we should just print the term we get from `convert_mod`